simulation/src/range_likelihood.cpp

```diff
@@ -180,336 +317,384 @@ pcl::simulation::RangeLikelihood::RangeLikelihood (int rows, int cols, int row_h
+pcl::simulation::RangeLikelihood::drawParticles(
+    std::vector<Eigen::Isometry3d, Eigen::aligned_allocator<Eigen::Isometry3d>> poses)
 {
   int n = 0;
-  for (int i=0; i<rows_; ++i)
-  {
-    for (int j=0; j<cols_; ++j)
-    {
-      glMatrixMode (GL_MODELVIEW);
-      glLoadIdentity ();
+  for (int i = 0; i < rows_; ++i) {
+    for (int j = 0; j < cols_; ++j) {
+      glMatrixMode(GL_MODELVIEW);
+      glLoadIdentity();
 
-      glViewport(j*col_width_, i*row_height_, col_width_, row_height_);
+      glViewport(j * col_width_, i * row_height_, col_width_, row_height_);
 
       // Go from Z-up, X-forward coordinate frame
       // to OpenGL Z-out,Y-up [both Right Handed]
       float T[16];
-      T[0] =  0;   T[4] = -1.0; T[8] =  0; T[12] = 0;
-      T[1] =  0;   T[5] = 0;    T[9] =  1; T[13] = 0;
-      T[2] = -1.0; T[6] = 0;    T[10] = 0; T[14] = 0;
-      T[3] =  0;   T[7] = 0;    T[11] = 0; T[15] = 1;
-      glMultMatrixf (T);
+      T[0] = 0;
+      T[4] = -1.0;
+      T[8] = 0;
+      T[12] = 0;
+      T[1] = 0;
+      T[5] = 0;
+      T[9] = 1;
+      T[13] = 0;
+      T[2] = -1.0;
+      T[6] = 0;
+      T[10] = 0;
+      T[14] = 0;
+      T[3] = 0;
+      T[7] = 0;
+      T[11] = 0;
+      T[15] = 1;
+      glMultMatrixf(T);
 
       // Apply camera transformation
-      applyCameraTransform (poses[n++]);
+      applyCameraTransform(poses[n++]);
 
       // Draw the planes in each location:
-      scene_->draw ();
+      scene_->draw();
     }
   }
 }
 
-
 /////////////////////////////////////////////////////////////////
 // Below are 4 previously used cost functions:
 // 0 original scoring method
 float
-costFunction0 (float ref_val,float depth_val)
+costFunction0(float ref_val, float depth_val)
 {
-  return  (sqr(ref_val - depth_val));
+  return (sqr(ref_val - depth_val));
 }
 
 // 1st working cost function:
 // Empirical reverse mapping between depthbuffer and true depth:
 // Version 0: [25 aug 2011]
 // TRUEDEPTH =  1/(1.33 -(DEPTHBUFFER)*1.29)
-//float cost = sqr(ref[col%col_width] - 1/(1.33 -(*depth)*1.29));
+// float cost = sqr(ref[col%col_width] - 1/(1.33 -(*depth)*1.29));
 // Version 1: [29 aug 2011] Exact version using correct mappings:
 float
-costFunction1 (float ref_val, float depth_val)
+costFunction1(float ref_val, float depth_val)
 {
-  float cost = sqr (ref_val - 1/(1.4285f - (depth_val)*1.3788f));
-  //std::cout << " [" << ref_val << "," << 1/(1.4285 -(depth_val)*1.3788) << "] ";
-  if (ref_val < 0)
-  { // all images pixels with no range
-    cost =1;
+  float cost = sqr(ref_val - 1 / (1.4285f - (depth_val)*1.3788f));
+  // std::cout << " [" << ref_val << "," << 1/(1.4285 -(depth_val)*1.3788) << "] ";
+  if (ref_val < 0) { // all images pixels with no range
+    cost = 1;
   }
-  if (cost > 10)
-  { // required to lessen the effect of modelpixel with no range (ie holes in the model)
+  if (cost > 10) { // required to lessen the effect of modelpixel with no range (ie
```

Comment:
Shifting the comment to next line would not break it and still maintain LOC count